data/remnant/remnant missions.txt

```diff
@@ -2231,3 +2243,678 @@ mission "Remnant: Face to Maw 2B"
+			`	"Thank you, Captain <first>. We have some ships that can follow, and there are other scouts already in the area that should be reporting soon. Given your experience with the wider galaxy, though, we would like you to lead the way."`
+				accept
+	on enter "Postverta"
+		dialog
+			`As your ship's sensors re-establish their depiction of the system around you, an alien space station comes into view. It appears to be old and mostly inactive, but your sensors do pick up occasional bursts of energy - and, oddly, faint signs of life almost everywhere.`
+	on stopover
+		conversation
+			`The station is surprisingly still active, despite its apparent age. Ancient environmental controls still maintain a breathable atmosphere, although it is thinner than you find comfortable. Likewise, the station is cool - but not dangerously so. Everywhere you look you continually feel like you have stumbled upon a space station built by giants: The doors are massive, controls tend to be fairly high up the walls, and displays seem to be angled for a view height significantly taller than yours. Most unusually, larger rooms often have what look to be benches and tables placed halfway up walls.`
+			`	The engineering of the station points to a dedicated construction program aimed at a single purpose. You trace some of the more obvious sets of cables and discover a large spherical chamber filled with machinery. Unfortunately, it is still emitting significant amounts of energy - enough that your suit's medical systems emphatically warn you to turn back. While the large sphere appears to be central to this station's purpose, it will have to remain a mystery for now. Clearly, this is the sort of thing that the Remnant sent you out to find.`
+	on complete
+		dialog
+			`As you arrive back at Viminal, you broadcast a warning to the port authorities that something is following you. You inform them that it is not hostile, but it is an unknown entity that originated in the new system.`
+
+
+
+mission "Remnant: Cognizance 15"
+	landing
+	invisible
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 14: active"
+	npc accompany
+		government "Ember Waste"
+		personality waiting mute
+		ship "Ember Waste Node" "Unknown Entity"
+	
+
+
+mission "Remnant: Cognizance 16"
+	name "Follow the Creature"
+	description "The creature seems to be creating constellations and trying to get you to follow it."
+	landing
+	source "Viminal"
+	waypoint
+		system Peragenor
+		system Convector
+		system Nenia
+	destination Nasqueron
+	to offer
+		has "Remnant: Cognizance 14: done"
+	on offer
+		conversation
+			`Quite a number of Remnant have gathered around the spaceport watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries are in position around the area in case the creature becomes violent.`
```

Comment:
```suggestion
			`Quite a number of Remnant have gathered around the spaceport watching the movements of the strange creature. It gestures wildly in various ways, then freezes in several different positions with the tips of certain tentacles glowing with energy. You note that several anti-air batteries around the area have their formidable-looking weapons trained on the creature in case it turns violent.`
```